packages/components/src/index.ts

```diff
@@ -63,3 +63,4 @@ export * from './Well'
 export * from './Workflow'
 export * from './Menu'
 export * from './Button'
```

Comment:
Whoops omg I didn't realise this auto-added itself when I moved things around. Could I trouble you to remove this for me? Since we're currently double exporting 😅 🙏🏻

Reply:
heheh nice, will axe it :)